### ISSUE TITLE
Device Status wrong linked

### ISSUE THREAD

**Issue by ace16vitamine** [jokob-sk/Pi.Alert#428]
Hi Jokob,

If I klick on Devices and then the Status of the Device the linked URL is wrong:

![Bildschirmfoto 2023-09-10 um 20 46 47](https://github.com/jokob-sk/Pi.Alert/assets/22064392/d6c25143-dd60-4e29-bce5-03332b4ae50a)

URL: http://192.168.1.20:20211/deviceDetails.php?mac=On-line

![Bildschirmfoto 2023-09-10 um 20 47 22](https://github.com/jokob-sk/Pi.Alert/assets/22064392/937e580d-3a81-4729-b687-a2c270080605)

**Comment by jokob-sk** [comment]
Hi there,

Can you let me know where did you get the `http://192.168.1.20:20211/deviceDetails.php?mac=On-line` link form? Where did you click?

Thanks,
j

**Comment by ace16vitamine** [comment]
See my initial Screenshots.

1) Click on Devices
2) Click on "Online"

**Comment by jokob-sk** [comment]
Thanks for that! 🙏

This is fixed in the `_dev` image - can you check?

**Comment by ace16vitamine** [comment]
I can confirm.

**Comment by jokob-sk** [comment]
keeping open so otehrs can find it

**Comment by jokob-sk** [comment]
Released in today's hotfix (23.9.11)

**Comment by zawier-git** [comment]
I have got similar situation. Version: v23.9.11 (hosted on docker).


```
Kernel:Linux

System:GNU/Linux

Operating System:Debian GNU/Linux 11 (bullseye)

Uname:Linux portainer 5.15.0-82-generic #91-Ubuntu SMP Mon Aug 14 14:14:14 UTC 2023 x86_64 GNU/Linux
Architecture:x86_64
```

![image](https://github.com/jokob-sk/Pi.Alert/assets/81773334/dd289673-7a9d-456a-b826-46e2bd4399c0)
That particular device is currently down. But interface shows it's online. 
![image](https://github.com/jokob-sk/Pi.Alert/assets/81773334/e891787c-91c5-4b4e-8820-90f0ed2b26f6)


While I do a scan:

`Starting Nmap 7.80 ( https://nmap.org ) at 2023-09-12 08:59 UTC
Note: Host seems down. If it is really up, but blocking our ping probes, try -Pn
Nmap done: 1 IP address (0 hosts up) scanned in 3.06 seconds`

In my case I feel like first scan of network just stucks and shows always the same info. 
Way I run Pi.Alert on docker is:

`docker run -d --network=host \
   --name=pialert \
  -v /home/zawier/Docker/PiAlert/config:/home/pi/pialert/config \
  -v /home/zawier/Docker/PiAlert/db:/home/pi/pialert/db \
  -e TZ=Europe/Warsaw \
  -e PORT=20211 \
  -e HOST_USER_ID=1000 \
  -e HOST_USER_GID=1000 \
  jokobsk/pi.alert:latest`

To be sure I have pulled from docker hub lastes version (which was updated yesterday).
